packages/db/package.json

```diff
@@ -64,6 +64,7 @@
     "@truffle/compile-common": "^0.8.1",
     "@truffle/contract-schema": "^3.4.10",
     "@truffle/resolver": "^9.0.16",
+    "@truffle/migrate": "^3.3.9",
```

Comment:
```suggestion
    "@truffle/migrate": "^3.3.10",
```

Reply:
Thanks!

Reply:
I'm really not sure how you managed to commit this, as the version check (see husky stuff in top-level package.json) should've caught it 🤔

Reply:
@eggplantzzz can you please confirm whether you committed with the `--no-verify` flag? If not, I'll raise a new issue for myself to go and fix that script.